services/cd-service/pkg/cmd/server.go

```diff
@@ -63,9 +66,51 @@ func (c *Config) readPgpKeyRing() (openpgp.KeyRing, error) {
 
 }
 
+// EnvConfig the server config that can be configured with env variables
+type EnvConfig struct {
+	//ProjectID       string `env:"GOOGLE_PROJECT_ID,required"`
+	EnvironmentName string `env:"ENV_NAME" envDefault:"local"`
+	TracingEnable   bool   `env:"TRACING_ENABLE" envDefault:"false"`
```

Comment:
either EnableTracing or TracingEnabled (with a `d` at the end) in both the fied name and the env name